docs/developer-guide.md

```diff
@@ -34,5 +34,6 @@ up your development environment in several ways:
 
 ## Formatting code using prettier
 
+* Using the Visual Studio Code editor - install the `prettier-vscode` extension
```

Comment:
Are you referring to `esbenp.prettier-vscode`? If so, specify the full name.

Reply:
Yeah, that's the package.

Within the extension ecosystem, `prettier-vscode` is a unique identifier for the package; the package can be installed with this command:

```
ext install prettier-vscode
```

I'm curious why Visual Studio Marketplace prefixes 'item' names with the name of the publisher.